spring-boot-project/spring-boot-docs/src/docs/antora/modules/reference/pages/messaging/pulsar.adoc

```diff
@@ -165,13 +165,13 @@ The following component creates a reactive listener endpoint on the `someTopic`
 include-code::MyBean[]
 
 Spring Boot auto-configuration provides all the components necessary for `ReactivePulsarListener`, such as the `ReactivePulsarListenerContainerFactory` and the consumer factory it uses to construct the underlying reactive Pulsar consumers.
-You can configure these components by specifying any of the `spring.pulsar.listener.*` and `spring.pulsar.consumer.*` prefixed application properties.
+You can configure these components by specifying any of the `spring.pulsar.listener.\*` and `spring.pulsar.consumer.*` prefixed application properties.
```

Comment:
This is an unrelated change that can easily be missed and the fix is in a different branch than this PR. Please don't do this.

I've fixed that in https://github.com/spring-projects/spring-boot/issues/42200